gui/default/syncthing/core/syncthingController.js

```diff
@@ -1296,6 +1296,7 @@ angular.module('syncthing.core')
 
         $scope.addFolder = function () {
             $scope.currentFolder = {
+                path: $scope.config.options.parentFolder,
```

Comment:
Just at first glance I think this might be a recipe for disaster as it will pass validation if the user just accepts the default. Do that more than once and you have multiple folders all syncing directly into to the `parentFolder`. If we're to suggest anything here, perhaps `$parentFolder/$folderID` or something like that.